src/Fhir.Anonymizer.Core.UnitTests/Extensions/SecurityTagExtensionsTests.cs

```diff
@@ -89,10 +85,7 @@ public void GivenAResourceWithSameSecurityLabels_WhenTryAddSecurityLabels_Securi
                     }
                 }
             };
-            var result = new ProcessResult()
-            {
-                IsRedacted = true
-            };
+            var result = new ProcessResult();
```

Comment:
Do we add "result.AddProcessRecord(AnonymizationOperations.Redact..." after this sentence? Default ProcessResult does not try to add any security labels.

Reply:
Sure, thanks for reminder.